test/typescript-karma/Dashboards/Synchronization/sync.js

```diff
@@ -10,10 +10,11 @@ Dashboards.DataGridPlugin.custom.connectDataGrid(DataGrid);
 Dashboards.PluginHandler.addPlugin(Dashboards.HighchartsPlugin);
 Dashboards.PluginHandler.addPlugin(Dashboards.DataGridPlugin);
 
-const { test } = QUnit;
+const { test, skip } = QUnit;
 
 
-test('Sync events leak in updated components', async function (assert) {
+// DataGrid1 Tests to update
+skip('Sync events leak in updated components', async function (assert) {
```

Comment:
The same as above.

Reply:
Tests for the DataGrid Component are also part of this task: https://app.asana.com/0/1203234853779882/1207620146790059/f